contrib/requirements.txt

```diff
@@ -0,0 +1,6 @@
+pytest
+pytest-env
+pytest-xdist
+
+# Timm
+git+https://github.com/rwightman/pytorch-image-models.git#egg=timm
```

Comment:
A challenge will be that different libraries might have conflicting versions of dependencies